src/main/java/uk/gov/crowncommercial/dts/scale/cat/controller/TendersController.java

```diff
@@ -132,4 +141,47 @@ public List<Release> getProjectsDeltas(
       
       return procurementEventService.getProjectUpdatesByLastUpdateDate(lastSuccessRun, jaggaerAPIConfig.getAssistedProcurementId() );
   }
+  
+  @GetMapping("/projects/rfxtemplatemapping/mappings")
+  @TrackExecutionTime
+  public ResponseEntity<List<RfxTemplateMapping>> getAllTemplateMappings(final ApiKeyAuthToken authentication) {
+    try {
+      List<RfxTemplateMapping> mappings = fileService.getAllTemplateMappings();
+      log.info("mappings: {}",mappings);
+
+      if (mappings.isEmpty()) {
+          log.info("mappings.IsEmpty() = TRUE");
+        return new ResponseEntity<>(HttpStatus.NO_CONTENT);
+      }
+
+      log.info("mappings: {}",mappings);
+
+      return new ResponseEntity<>(mappings, HttpStatus.OK);
+    } catch (Exception e) {
```

Comment:
Generally you don't need to catch the exceptions unless you specifically want to return a specific error message. Best to let the exceptions propagate out of this method where they will be caught and handled by a generic/application wide exception handler and then handled in a consistent way. You should just be able to remove the try/catch and add the exceptions to the method. If you are catching checked exceptions here then you could catch and then rethrow a runtime exception instead which will then be handled by the exception handler.

I haven't looked to see how this is handled in other parts of the application - but thats the way I would expect it to be done.